frontend/ui/src/app/deployments/deployment-target-card/deployment-target-metrics.component.html

```diff
@@ -1,13 +1,11 @@
-@let cpu = getPercent(metrics().cpuUsage);
-@let memory = getPercent(metrics().memoryUsage);
 <div class="flex space-x-1 items-center justify-end rtl:space-x-reverse">
   <div class="flex flex-col justify-center items-center">
     <div class="relative">
-      <div class="gauge z-0 percent-{{ cpu }} relative">
+      <div class="gauge z-0 percent-{{ getPercentClass(metrics().cpuUsage) }} relative">
         <div class="gauge-center bg-white dark:bg-gray-800"></div>
       </div>
       <div class="absolute inset-0 flex items-center justify-center text-xs font-thin text-black dark:text-white">
-        {{ cpu }}%
+        {{ getPercent(metrics().cpuUsage) }}%
```

Comment:
OK sorry for another "one more thing" :see_no_evil: but I think at this point it would make sense to use angulars built-in `percent` pipe for this: https://angular.dev/api/common/PercentPipe